analysis/src/bin/analysis-driver.rs

```diff
@@ -61,13 +61,13 @@ fn get_attribute<'tcx>(
                         },
                     args: ast::MacArgs::Empty,
```

Comment:
This has been here for a while, so the `analysis` crate only sees `prusti` attributes without arguments/values. Is this intended, or even needed @fpoli ?